win32ss/gdi/gdi32/objects/font.c

```diff
@@ -460,23 +460,24 @@ GetCharacterPlacementW(
     /* return number of initialized fields */
     lpResults->nGlyphs = nSet;
 
-    /*if((dwFlags&GCP_REORDER)==0 || !BidiAvail)
-      {*/
-    /* Treat the case where no special handling was requested in a fastpath way */
-    /* copy will do if the GCP_REORDER flag is not set */
-    if(lpResults->lpOutString)
-        lstrcpynW( lpResults->lpOutString, lpString, nSet );
+    if((dwFlags&GCP_REORDER)==0 )
+    {
+        /* Treat the case where no special handling was requested in a fastpath way */
+        /* copy will do if the GCP_REORDER flag is not set */
+        if(lpResults->lpOutString)
+            lstrcpynW( lpResults->lpOutString, lpString, nSet );
 
-    if(lpResults->lpOrder)
+        if(lpResults->lpOrder)
```

Comment:
Add one space between `if` and its opening parenthesis (the three "if"s above). Same remark also for the for-loop below.